Python/PR_Analyzer/main.py

```diff
@@ -0,0 +1,381 @@
+#Necessary Packages
+import datetime
+import json
+import re
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+from jupyterthemes import jtplot
+from matplotlib import pyplot as plt
+from sklearn.cluster import KMeans
+
+jtplot.style('monokai','notebook',ticks=True)
+
+
+#---------------------------------------------------------------------------------------------------------#
+
+"""
+This Section contains code block to fetch all the Pull Requests along with title.
+The result is zipped to a Pandas DataFrame.
+"""
+#Code block to fetch the all Pull Requests along with their links.
+prs=[]
+no=[]
+info=[]
+pr_title=[]
+for x in range(1,30):
+    r=requests.get(url="https://github.com/HarshCasper/Rotten-Scripts/pulls?page="+str(x)+"&q=is%3Apr")
```

Comment:
Use a dynamic repo. You can use an argument parser to ask the user to pass the repo to the script.

Reply:
Sure I will make it right away.